### ISSUE TITLE
View task in browser [modifier]

### ISSUE THREAD

**Issue by melikarslan** [moranje/alfred-workflow-todoist#34]
I don't know if this is the correct place to ask features, if not please let me know so I can move/delete. 

- Option to view task in browser instead of marking as complete when pressing enter. I don't know if this is technically possible, but it would be more than great if it opens in the Todoist app (if installed).

- Adding comments & files to created tasks. 

Again, thanks for great work @moranje !

**Comment by infn8** [comment]
+1 on this.  My thought is that either <option> + <enter> or <ctrl> + <enter> would be a good candidate.

**Comment by moranje** [comment]
Let's track adding support for comments in #31. Opening in the browser should be possible with modifier keys, but I'm unsure how. If anyone is willing to dig in to the Alfred documentation, I'm willing to implement this if it's not to much work.

**Comment by melikarslan** [comment]
Thanks for supporting this @moranje.

Two things:

1- Will you consider making it default instead of triggering with a modifier? When I was realized pressing enter makes the task completed, I was quite shocked. Because, I was expecting to browse the task, since we all developed intuition over the years to see details of the item when pressing enter, not trashing it. 

2- I have one more feature suggestion about this: After creating a task, Alfred sends a notification that it has been successfully created. It would be AWESOME to open the task in Todoist app when clicking that notification. Since you already said that adding comments feature won't be coming on next release, having this option would help us to add comments quickly and easily.

**Comment by moranje** [comment]
I've looked into a notification module outside of alfred but I'm afraid I can't get it to work with the bundler I am using. Alfreds built-in notifications won't allow for clicking a link.

As for 1. the task does say 'COMPLETE TASK' so it should have come as a complete suprise. I rarely feel the need to edit items after I created them, I have also not heard the request before so I am going to leave completion the default option. If there is an overwhelming response from other people I will consider it but I would suggest tracking that in a separate issue.
